internal/locationutil/location.go

```diff
@@ -0,0 +1,52 @@
+package locationutil
+
+import (
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/values"
+)
+
+type Location struct {
+	Name   string
+	Offset flux.Duration
+}
+
+func (l Location) IsUTC() bool {
+	name := l.Name
+	if name == "" {
+		name = "UTC"
+	}
+	return name == "UTC" && l.Offset.IsZero()
+}
+
+func (l Location) Load() (LocationZone, error) {
+	name := l.Name
+	if name == "" {
+		name = "UTC"
+	}
+	loc, err := LoadLocation(name)
+	if err != nil {
+		return LocationZone{}, err
+	}
+	loc.Offset = l.Offset
+	return loc, nil
+}
+
+func (l Location) ToUTCTime(t values.Time) values.Value {
```

Comment:
`ToUTCTime` isn't a good name as it doesn't describe what this function does. We've avoided using the name `UTC` in the timezone code as to not confuse it with `UTC` as a concept.